app/models/type.rb

```diff
@@ -72,9 +72,9 @@ def <=>(type)
     name <=> type.name
   end
 
-  def self.all
-    find(:all, order: 'position')
-  end
+  # def self.all
+  #  find(:all, order: 'position')
+  # end
```

Comment:
@NobodysNightmare if this PR were smaller and more manageable, then I'd probably rebase the comments out.